streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -467,7 +467,7 @@ public void shouldUseSpecifiedNameForSinkProcessor() {
         stream.to(STREAM_TOPIC_TWO);
         builder.build();
         final ProcessorTopology topology = builder.internalTopologyBuilder.rewriteTopology(new StreamsConfig(props)).build();
-        assertSpecifiedNameForOperation(topology, "KSTREAM-SOURCE-0000000000", expected, "KSTREAM-SINK-0000000001");
+        assertSpecifiedNameForOperation(topology, "KSTREAM-SOURCE-0000000000", expected, "KSTREAM-SINK-0000000002");
```

Comment:
The first source is `KSTREAM-SOURCE-0000000000` second would be `...01` so this change is correct.